src/patternfly/components/Form/docs/code.md

```diff
@@ -1,18 +1,14 @@
-## Overview
-
-Only if needed write a short description with implementation notes. Design and interaction notes are already written on the design specs, don't repeat information.
-
 ## Accessibility
 
 | Attribute | Applied To | Outcome |
 | -- | -- | -- |
-| `for` | `<label id="{for}">` |  Each `<label>` must have a `for` attribute that matches its form field id. **Required** |
-| `id` | `all form inputs, radios, checkboxes, selects, textareas...` |  Each `<form>` field must have an `id` attribute that matches its labels `for` value. **Required** |
-| `data-required="true"` | `<label` |  Labels associated with required fields must include this attribute. |
-| `required aria-required="true"` | `<input>` | Required fields must include these attributes. |
+| `for` | `<label>` |  Each `<label>` must have a `for` attribute that matches its form field id. **Required** |
+| `id` | `<input type="radio/checkbox/text">`, `<select>`, `<textarea>` |  Each `<form>` field must have an `id` attribute that matches its label's `for` value. **Required** |
+| `data-required="true"` | `<label>` |  Labels associated with required fields must include this attribute. |
```

Comment:
I'm not familiar with why `data-required` is here. Is this something we need? And if if so, should we include a statement to clarify what it's purpose is? I looked at the current version, but don't see anything that's tied to this attribute.

cc @mattnolting on this one

Reply:
Not sure where I got data-required. It doesn't appear to have any value.